src/cli/utils/tests/async-load-config.spec.ts

```diff
@@ -0,0 +1,43 @@
+import { readFileSync, unlinkSync, writeFileSync } from 'fs';
+import { asyncLoadConfig } from '../async-load-config';
+
+const pathToConfigFile = `${process.cwd()}/medusa-config.js`;
+const runLoadTest = async (testFixtureFileName: string) => {
```

Comment:
You can rename the method to getConfig and use it in the test case, each test case should be able to explicitly check the values of each params of the fake config and not just instance of promise etc since they should be resolved already

Reply:
this line does it expect(configModule.projectConfig).toMatchObject(testProjectConfig);